ControlRoom/Controllers/SimCtl+SubCommands.swift

```diff
@@ -0,0 +1,745 @@
+        }
+        /// Copy standard input onto the device pasteboard.
+        static func pbcopy(device: Pasteboard.Device, flags: [Pasteboard.Flag] = []) -> Command {
+            Command(.pbcopy, arguments: device.arguments + flags.flatMap { $0.arguments })
+        }
+        /// Print the contents of the device's pasteboard to standard output.
+        static func pbpaste(device: Pasteboard.Device, flags: [Pasteboard.Flag] = []) -> Command {
+            Command(.pbpaste, arguments: device.arguments + flags.flatMap { $0.arguments })
+        }
+        /// Set up a device IO operation.
+        static func io(deviceId: String, operation: IO.Operation) -> Command {
+            Command(.io, arguments: operation.arguments)
+        }
+        /// Collect diagnostic information and logs.
+        static func diagnose(flags: [Diagnose.Flag]) -> Command {
+            Command(.diagnose, arguments: flags.flatMap { $0.arguments })
+        }
+        /// enable or disable verbose logging for a device
+        static func logverbose(deviceId: String?, isEnabled: Bool = false) -> Command {
+            var arguments: [String] = []
+            if let deviceId = deviceId {
+                arguments.append(deviceId)
+            }
+            return Command(.logverbose, arguments: arguments + [(isEnabled ? "enabled" : "disabled")])
+        }
+        /// Set or clear status bar overrides
+        static func statusBar(deviceId: String, operation: StatusBar.Operation) -> Command {
+            Command(.statusBar, arguments: [deviceId] + operation.arguments)
+        }
+        /// Get or Set UI options
+        static func ui(deviceId: String, option: UI.Option) -> Command {
+            Command(.ui, arguments: [deviceId] + option.arguments)
+        }
+        /// Send a simulated push notification
+        static func push(deviceId: String, appBundleId: String? = nil, json: Push.JSON) -> Command {
+            var arguments: [String] = [deviceId]
+            if let appBundleId = appBundleId {
+                arguments.append(appBundleId)
+            }
+            return Command(.push, arguments: arguments + json.arguments)
+        }
+        /// Grant, revoke, or reset privacy and permissi Manipulate a device's keychain
+        static func privacy(deviceId: String, action: Privacy.Action, service: Privacy.Permission, appBundleId: String? = nil) -> Command {
+            var arguments: [String] = [deviceId] + action.arguments + service.arguments
+            if let appBundleId = appBundleId {
+                arguments.append(appBundleId)
+            }
+            return Command(.privacy, arguments: arguments)
+        }
+        /// Manipulate a device's keychain
+        static func keychain(deviceId: String, action: Keychain.Action) -> Command {
+            Command(.keychain, arguments: [deviceId] + action.arguments)
+        }
+    }
+
+    enum SubCommand: String {
```

Comment:
I'm not convinced this enum is necessary. We could just put the subcommand string directly in the static funcs

Reply:
yeah, definitely less running around to make changes without it 👍